CHANGELOG.md

```diff
@@ -1,20 +1,48 @@
-## 3.3.0 2024-06-13
-[GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
-## Unreleased 2021-11-10
-[GP-2876](https://jira.oicr.on.ca/browse/GP-2876) - make calculate.sh script more robust with md5sum calculation
-## 3.2.0 2021-09-10
-[GC-8937](https://jira.oicr.on.ca/browse/GC-8937) - add parameters to better control Java Heap Memory
-## 3.1.2 2021-05-31
-[GP-2733](https://jira.oicr.on.ca/browse/GP-2733) - migrate to Vidarr
-## 3.1.2 2021-05-26
-[GP-2726](https://jira.oicr.on.ca/browse/GP-2726) - remove optional output for read 1
-## 3.1.1 2020-01-29
-[GP-2259](https://jira.oicr.on.ca/browse/GP-2259) - adding memory setting to the last job (previously missing)
-## 3.1 - 2020-01-12
-Just a version increment, to avoid collision with previous installation. Added docs
-## 3.0 - 2019-09-05
-[GP-2088](https://jira.oicr.on.ca/browse/GP-2088) - Converting workflow to wdl, parent pom 2.5.7
-## 2.5 - 2016-03-31
-[GP-760](https://jira.oicr.on.ca/browse/GP-760) - Update workflow-fastqc to provision out zip file and update to version 0.11.5.
-## 2.4 - 2015-03-19
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [3.3.0] 2024-06-13
+### Added
+- [GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - Add vidarr labels to outputs (changes to medata only).
+
+## [Unreleased] 2021-11-10
+### Changed
+- [GP-2876](https://jira.oicr.on.ca/browse/GP-2876) - Make calculate.sh script more robust with md5sum calculation.
+
+## [3.2.0] 2021-09-10
+### Added
+- [GC-8937](https://jira.oicr.on.ca/browse/GC-8937) - Add parameters to better control Java Heap Memory.
+
+## [3.1.2] 2021-05-31
+### Changed
+- [GP-2733](https://jira.oicr.on.ca/browse/GP-2733) - Migrate to Vidarr.
+
+## [3.1.2] 2021-05-26
+### Removed
+- [GP-2726](https://jira.oicr.on.ca/browse/GP-2726) - Remove optional output for read 1.
+
+## [3.1.1] 2020-01-29
+### Added
+- [GP-2259](https://jira.oicr.on.ca/browse/GP-2259) - Adding memory setting to the last job (previously missing).
+
+## [3.1] - 2020-01-12
```

Comment:
This hyphen between the [version] and the date is a part of format, please make sure it is used consistently everywhere